src/unwinder/find_fde/mod.rs

```diff
@@ -35,6 +37,10 @@ impl FDEFinder for GlobalFinder {
         if let Some(v) = fixed::get_finder().find_fde(pc) {
             return Some(v);
         }
+        #[cfg(feature = "fde-gnu-eh-frame-hdr")]
+        if let Some(v) = gnu_eh_frame_hdr::get_finder().find_fde(pc) {
+            return Some(v);
+        }
```

Comment:
Should this perhaps be moved up so that it's searched before PHDR?

Reply:
Sure.